internal/local/command_kafka_start.go

```diff
@@ -115,10 +117,13 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	output.Println("\r")
-
 	log.CliLogger.Tracef("Successfully pulled Confluent Local image")
 
-	if err := c.prepareAndSaveLocalPorts(cmd, c.Config.IsTest); err != nil {
+	numOfBrokers, err := cmd.Flags().GetInt32("brokers")
```

Comment:
Based on the `// range: [1, 4]` comment, should that limit be enforced here? At a quick glance I didn't see anything that seems to stop users from choosing higher numbers.

Reply:
that's right, no enforcement added yet!